tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/BootstrapTasks/Microsoft.Azure.Build.BootstrapTasks.csproj

```diff
@@ -18,7 +19,6 @@
     <TaskBinaryOutput>..\..\..\tasks</TaskBinaryOutput>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFramework>net46</TargetFramework>
```

Comment:
Where is TargetFramework getting set to net46 now?

Reply:
I don't quite follow the "where" part.
If you are wondering why are we targeting .NET 46, then all our build tools are targeting net 46 and above.